beetsplug/replaygain.py

```diff
@@ -104,6 +104,8 @@ class Backend(object):
     """An abstract class representing engine for calculating RG values.
     """
 
+    do_parallel = False
```

Comment:
I don't know if it's dangerous dangerous, but I just haven't been able to get `gstreamer` to work in parallel with `ThreadLoop`. It looses track of threads non-determenistically; sometimes it writes one (or a couple) of the files, sometimes it throws 
![image](https://user-images.githubusercontent.com/31547038/73382356-a1e0a280-42c7-11ea-9e06-11d714514d4c.png) 
and sometimes it just fails silently. 

And for now, I haven't been able to test it with `audiotools` since it doesn't want to install on my machine for some reason.

I'm going to look into both of these issues, but I wanted to see if the whole thing would be useful first.

Reply:
Sound good! I think it's fine to ship this feature before we figure out how to parallelize GStreamer.